rubocop-rspec.gemspec

```diff
@@ -39,4 +39,5 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'rubocop', '~> 1.33'
   spec.add_runtime_dependency 'rubocop-capybara', '~> 2.17'
+  spec.add_runtime_dependency 'rubocop-factory_bot'
```

Comment:
Lock to a specific version? See #1567

Reply:
We can't lock it down just yet, so I'm leaving it in the comments.

Reply:
What would happen if you lock it here?

Reply:
I got an error that such a version does not exist, but I may be forgetting something.